README.md

```diff
@@ -99,7 +128,9 @@ $("button").on("click", function(event){
 
 Your Answer:
 ```js
-// write code here
+document.querySelectorAll('button', function(event){
+  event.target.addEventListener(onclick).css('background-color', 'red')
+})
 ```
```

Comment:
Close, but with vanilla JS when `.querySelectorAll` returns a collection of DOM elements, you need to iterate through each item individually to give them each an event listener. So it would look something like...

```js
var buttons = document.querySelectorAll("button");
for(var i = 0; i < buttons.length; i++){
  buttons[i].addEventListener("click", function(evt){
    evt.target.style.color = "red";
 })
};
```